### PR TITLE
SEO

### PR DESCRIPTION
**Pull request by rezrah**
See changed files for changeset items, and additions to Storybook.

Refer to [Slack 🧵 for more info. ](https://github.slack.com/archives/C04E44ZPNEA/p1715351753976649)

## What should reviewers focus on?

- Changes to components
- Reviewing templates for correctness

### CODE REVIEW

File: .changeset/piano-hat-priority [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
```
Comment (joshfarrant) [inline .changeset/piano-hat-priority:1]: Unsure if it matters, but this filename is missing the `.md` file extension

**Review by joshfarrant — APPROVED** [review]
Looks great! 🚀

One tiny thing regarding the changesets, but otherwise lgtm!

File: .changeset/piano-hat-priority (hunk 1 above)
Comment (rezrah) [reply .changeset/piano-hat-priority:1]: Thanks @joshfarrant

File: packages/react/src/Card/Card.module.css [hunk 2]
```diff
@@ -78,7 +86,7 @@
   align-self: center;
 }
 
-.Card--colorMode-light:not(.Card--disableAnimation):hover {
+.Card:not(.Card--disableAnimation, .Card--variant-default.Card--colorMode-dark):hover {
   transform: scale3d(1.025, 1.025, 1);
```
Comment (nsolerieu) [inline packages/react/src/Card/Card.module.css:90]: @rezrah can we use a more subtle opacity fade instead?

It's looking very bouncy, too playful. The easing is fine but I'd suggest removing the `scale3d`

File: packages/react/src/recipes/seo/Category/CategoryPage.tsx [hunk 3]
```diff
@@ -0,0 +1,295 @@
...
+              paddingBlockEnd={{narrow: 48}}
+              borderBlockEndWidth="thin"
+              borderColor="default"
+              borderStyle="solid"
+            >
+              <Grid enableOverlay={enableGridOverlay} className={styles.cardGrid}>
+                {Array.from({length: 6}).map((_, index) => (
+                  <Grid.Column span={{xsmall: 12, medium: 6}} key={index}>
+                    <ThemeProvider colorMode={selectedColorMode}>
+                      <Box animate="scale-in-right">
```
Comment (nsolerieu) [inline packages/react/src/recipes/seo/Category/CategoryPage.tsx:234]: @rezrah can we use a more subtle `fade-in` here?

**Review by nsolerieu — APPROVED** [review]
Minor animation notes - otherwise LGTM
All the major `Storybook QA` in [figma](https://www.figma.com/file/9geH9GfR3y6TH8n4HA6sJC/Resources-migration-%2B-SEO-Templates?type=design&node-id=1431%3A24112&mode=design&t=JAy2Hn4iqN3uM4UC-1) has been covered

File: packages/react/src/Card/Card.module.css (hunk 2 above)
Comment (rezrah) [reply packages/react/src/Card/Card.module.css:90]: This is the current default for cards, so making this change would change all light mode cards globally. Is that something we can defer as we'd need more time to investigate impact.

**Review by simmonsjenna — APPROVED** [review]
Looks good! 👍

**Pull request merged**